packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -94,3 +97,20 @@ export const validateAlphabeticalCharacters = (name: string): boolean => {
 export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
+
+/**
+ * A function which returns hash code for the avatar color
+ *
+ * @param str is idForColor or name
+ * @returns hash code of the color
+ */
+const getHashCode = (str: string): number => {
```

Comment:
We may want to put this in its own file so that we can have per-platform implementations of this function.